Extension/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 1.11.0: June 20, 2022
+### New Features
+* Add inlay hints for parameters and auto types. [#5845](https://github.com/microsoft/vscode-cpptools/issues/5845)
```

Comment:
Could you add [the settings PR](https://github.com/microsoft/vscode-cpptools/pull/9460) for inlay hints so that the settings get referenced in the change log?

Reply:
Actually, I'll just reference the PR changes in the issue.